app/views/request-to-go-live/index.njk

```diff
@@ -8,17 +8,10 @@
 
 {% block mainContent %}
   {% if currentService.currentGoLiveStage === goLiveStage.LIVE %}
-    <div class="govuk-grid-column-full">
+    <div class="govuk-grid-column-full govuk-!-margin-top-6">
       <div class="next-steps-panel">
```

Comment:
What is this div for?

Reply:
As in the one with `.next-steps-panel`

Reply:
it’s not used in tests AFAIK and if it was in could be added to the parent div rather than adding a whole new one

Reply:
Thanks, we tried the suggestion, looks like when we apply the changes the `next-steps-panel` is bigger than expected, also https://design-system.service.gov.uk/components/error-summary/ generates internal div as well, we can show you visually the changes and let's discuss when you are available